test/util/cart_utils.py

```diff
@@ -231,6 +242,9 @@ def build_cmd(self, cartobj, env, host, report_uri=True, urifile=None):
         if tst_env is not None:
             tst_cmd += " " + tst_env
 
+        if logparse:
+	    tst_cmd += "-x D_LOG_FILE_APPEND_PID=1"
```

Comment:
You probably want a space at the beginning for consistency.
" -x D_LOG_FILE_APPEND_PID=1"
You will run into potential cmd error without the space, say if you define tst_env in your yaml and it doesn't end with a space.

Reply:
okay